demisto_sdk/commands/common/tools.py

```diff
@@ -2223,9 +2236,113 @@ def is_test_config_match(
     if test_playbook_id:
         return test_playbook_match
 
+    if script_id:
+        return scripts_match
+
     return False
 
 
+def is_content_item_dependent_in_conf(test_config, file_type) -> bool:
+    """Check if a line from conf have multiple integration/scripts dependent on the TPB.
+        - if the TPB checks only one integration/script it is independent.
+        - if the TPB checks more then one integration/script it is dependent.
+    Args:
+        test_config (dict): The dict in the conf file.
+        file_type (str): The file type, can be integrations, scripts or playbook.
+
+    Returns:
+        bool: The return value. True for dependence, False otherwise.
+    """
+    integrations_list = test_config.get("integrations", [])
+    integrations_list = (
+        integrations_list
+        if isinstance(integrations_list, list)
+        else [integrations_list]
+    )
```

Comment:
```suggestion
integrations_list = integrations_list if isinstance(integrations_list, list) else [integrations_list]
```

Reply:
No need for all the extra lines here, ignore the black if necessary
same for scripts